packages/commonwealth/client/scripts/views/pages/manage_community/manage_roles.tsx

```diff
@@ -112,7 +112,7 @@ export const ManageRoles = ({
           onClick: async () => {
             await removeRole(role);
             if (isLosingAdminPermissions) {
-              navigate(`/`);
+              navigate(`/manage/moderators`);
```

Comment:
lets update this to `navigate(featureFlags.newAdminOnboardingEnabled ? '/manage/moderators' : '/')`